src/CircleGraph/CircleGraphCtrl.ts

```diff
@@ -55,6 +55,7 @@ export class MessageDefs {
   public static readonly backendColor = "backendColor";
   public static readonly error = "error";
   public static readonly colorTheme = "colorTheme";
+  public static readonly scrollToSelected = "scrollToSelected";
```

Comment:
I cannot accept this change and with related changes from the draft.
I need to test other modules with your change in the draft but cannot do the test as I am busy with other works and I cannot say when.

If possible, please do not fix this module but fix your codes.
If not, please post a new issue what you want, what the problem is with current codes.

Reply:
@seanshpark 
I see. Thank you. ok.

Reply:
I've reviewed your draft code and there's no modification to existing features so I think it's OK to land the change.

Reply:
Ahh. ok. Thank you!